gst/nnstreamer/tensor_query/tensor_query_client.c

```diff
@@ -192,6 +198,8 @@ gst_tensor_query_client_init (GstTensorQueryClient * self)
   self->timeout = DEFAULT_CLIENT_TIMEOUT;
   self->edge_h = NULL;
   self->msg_queue = g_async_queue_new ();
+  self->drop = FALSE;
```

Comment:
How about using the default value that you defined?
```suggestion
  self->drop = DEFAULT_DROP;
```

Reply:
Thanks, updated!